.github/workflows/bot.yml

```diff
@@ -9,6 +9,10 @@ on:
     branches:
       - master
       - 'release-*'
+    paths-ignore:
+      - '.github/**'
+      - '**.md'
+      - '**.py'
```

Comment:
can you pls clarify what this setting implies?

Reply:
Not sure if this is what should be in the final version, and it should be in a separate pr probably. What it does is if you only modify a something that matches those, the rest of the unit tests won't run. I think it definitely needs to change because if we update bot.yml it should run the tests, but if someone just makes changes to a readme we don't need the tests to re-run